scheduler/src/cook/mesos/api.clj

```diff
@@ -741,7 +741,7 @@
                  :max_retries (:job/max-retries job) ; consistent with input
                  :retries_remaining (- (:job/max-retries job) (util/job-ent->attempts-consumed db job))
                  :max_runtime (:job/max-runtime job Long/MAX_VALUE) ; consistent with input
-                 :framework_id fid
+                 :framework_id (str fid)
```

Comment:
Ok I gave up on this one. I have no idea how this is getting converted, so I made a special case in the test.